packages/plugins/rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec

```diff
@@ -19,7 +19,6 @@ Requires: %{?scl_prefix}rubygem(foreman-tasks-core) < 1.0
 Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) >= 1.1
 Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) < 2.0
 Requires: %{?scl_prefix}rubygem(net-ssh) >= 4.0
```

Comment:
this still lists >= 4, I think the part `>= 4.0` should be removed, that sould keep the dependency without version restriction